src/browser/modules/Stream/CypherFrame/helpers.test.ts

```diff
@@ -648,6 +648,31 @@ describe('helpers', () => {
       // Then
       expect(res).toEqual([['"P1M2DT3.000000004S"']])
     })
+
```

Comment:
do we test the records that come from:
return "hello \" "
or 
return '"'

Reply:
=> hello " ,
=> ",

=> "
""
"

Maybe it should become something like those?

Reply:
Tried these ones and the results end up looking like:
1. `return "hello \" "`:
```
"""hello \"" """
"hello \"" "
```

2. `return '"'`:
```
"'""'"
"\"""
```

3. `return '
"
'` (with new lines):
```
"'
""
'"
"
\""
"
```

Seems like they are all valid csv exports but let me check why in the first case the column name got triple quotes once again instead of just one.

Reply:
Also tried the below query to see different types as we've talked before.
```
CREATE (n {string: "string",boolean: true,integer: 1,float: 2.0,point: point({x:1,y:2}),date: date(),time: time(),datetime: datetime(),localtime: localtime(),localdatetime:localdatetime(),duration: duration("P5M1.5D"),numberList: [1,2,3],stringList: ["a"],emptyList: []}) RETURN n, n.string, n.boolean, n.integer, n.float, n.point, n.date, n.time, n.datetime, n.localtime, n.localdatetime, n.duration, n.numberList, n.stringList, n.emptyList
``` 
The exported CSV looks as follows:
```
n,n.string,n.boolean,n.integer,n.float,n.point,n.date,n.time,n.datetime,n.localtime,n.localdatetime,n.duration,n.numberList,n.stringList,n.emptyList
"({localtime: 10:21:41.776000000,date: 2024-06-06,string: string,integer: 1,float: 2.0,point: point({srid:7203, x:1, y:2}),duration: P5M1DT43200S,emptyList: [],datetime: 2024-06-06T10:21:41.776000000Z,boolean: true,stringList: [a],numberList: [1, 2, 3],time: 10:21:41.776000000Z,localdatetime: 2024-06-06T10:21:41.776000000})",string,true,1,2.0,"point({srid:7203, x:1, y:2})",2024-06-06,10:21:41.776000000Z,2024-06-06T10:21:41.776000000Z,10:21:41.776000000,2024-06-06T10:21:41.776000000,P5M1DT43200S,"[1, 2, 3]",[a],[]
```
which is a valid CSV without triple quotes 💃.

Thinking back on the triple quotes in the column name in the top comment, I believe it is expected as indeed the column name is `"hello \" "` rather than just `hello \" `. We can try to remove the quotes in that case as well but for that, we may need to add an if check which removes the quotes from the beginning and the end of the result record.

Will be testing the jar file and merge the pr if all is good.